src/main/webapp/mapScript.js

```diff
@@ -163,33 +186,41 @@ async function createRoute() {
       },
     };
 
-    // TODO(#17): Handle response from fetch.
     await fetch("/storeRoute", options)
       .then((response) => response.json())
-      .then((status) => {
-        if (status.hasOwnProperty("errorMessage")) {
-          alert(status.errorMessage);
+      .then((jsonResponse) => {
+        if (jsonResponse.hasOwnProperty("errorMessage")) {
+          alert(jsonResponse.errorMessage);
         } else {
-          console.log("receive new place's id " + status.toString());
+          if (jsonResponse.isPublic === true) {
```

Comment:
is it possible to just use `if (jsonResponse.isPublic) {` ?

Reply:
Sure, thanks